ospfd/ospf_flood.c

```diff
@@ -129,6 +128,22 @@ struct external_info *ospf_external_info_check(struct ospf *ospf,
 		}
 	}
 
+	if (is_prefix_default(&p) && ospf->external[DEFAULT_ROUTE]) {
```

Comment:
'zclient->default_information' flag need to be validated  before redistributing this route , right ?

Reply:
**NAK**

`ospf->external[DEFAULT_ROUTE]` is `default-information originate always`.  It's specifically for the _always_ part.